engines/director/detection_tables.h

```diff
@@ -2612,6 +3085,15 @@ static const DirectorGameDescription gameDescriptions[] = {
 	// Original filename is ラインの黄金CD
 	MACDEMO1_l("rheingold", "Demo", "Das Rheingold CD", "b7e69c37b7355022d400c14aa97c5d54", 484351, Common::JA_JPN, 404),
 
+    // Developed by Masahiro Fukuda & Co, published by Mizuki of Kashiba, Nara, Japan (1995)
```

Comment:
And in this block. Please, doublecheck the whole file.

Reply:
Yes, I think I did a big find/replace because I didn't think to check the style guide first! 🤦

I'll fix this.

Reply:
Done.